src/PlanetData.js

```diff
@@ -25,49 +44,81 @@ PlanetData.prototype = {
     this.mapData.canvas.width = this.width;
     this.mapData.canvas.height = this.height;
     this.mapData.ctx = this.mapData.canvas.getContext('2d');
-    this.mapData.imageData = this.mapData.ctx.getImageData(0,0,this.mapData.canvas.width, this.mapData.canvas.height);
+    this.mapData.imageData = this.mapData.ctx.getImageData(0, 0, this.mapData.canvas.width, this.mapData.canvas.height);
     this.mapData.data = this.mapData.imageData.data;
 
-    // TODO: add hydro erosion to this
-    // TODO: add complex features, cliffs, rivers, etc.
     //create a 2d array which holds our height data for the planet
     //This is done so that we can avoid calling our noise function more than once per pixel
     var buffer = [];
-    for (var x = 0; x < this.width; x++) {
+    var rivers = [];
+    var peak = 0;
+    var low = 1;
+    var terh = 0;
+    var rv = 0;

```

Comment:
These two variables would be better off with their full name instead of abbreviated I think.